src/main/filesystem/markdown.js

```diff
@@ -107,6 +109,26 @@ export const loadMarkdownFile = async (pathname, preferedEol, autoGuessEncoding
     markdown = convertLineEndings(markdown, 'lf')
   }
 
+  // Detect final newline
+  if (trimTrailingNewline === 2) {
+    if (!markdown) {
+      // Use default value
+      trimTrailingNewline = 3
+    } else {
+      const lastIndex = markdown.length - 1
+      if (lastIndex >= 1 && markdown[lastIndex] === '\n' && markdown[lastIndex - 1] === '\n') {
+        // Disabled
+        trimTrailingNewline = 2
+      } else if (markdown[lastIndex] === '\n') {
+        // Ensure single trailing newline
+        trimTrailingNewline = 1
+      } else {
+        // Trim trailing newlines
+        trimTrailingNewline = 0
```

Comment:
>Ensure a single trailing newline or whether trailing newlines should be removed: `0`: ensure single newline, `1`: trim all trailing newlines, `2`: auto detect, `3`: disabled. 

Why `trimTrailingNewline` is 0(ensure single newline) when there are no `\n` at the end of markdown, I think it should be 1(trim all trailing newlines)?

Reply:
Because the documentation is wrong.